--- a/backend/Services/Storage/Filesystem.php
+++ b/backend/Services/Storage/Filesystem.php
@@ -227,8 +227,8 @@ class Filesystem implements Service
     {
         $adapter = $this->storage->getAdapter();
         
-        switch (get_class($adapter)) {
-            case 'League\Flysystem\Adapter\Local':
+        switch (true) {
+            case $adapter instanceof \League\Flysystem\Adapter\Local:
                 $absolutePath = $adapter->applyPathPrefix($path);
                 return chmod($absolutePath, octdec($permissions));
                 break;
@@ -288,8 +288,8 @@ class Filesystem implements Service
         $adapter = $this->storage->getAdapter();
         $path = $entry['path'];
         
-        switch (get_class($adapter)) {
-            case 'League\Flysystem\Adapter\Local':
+        switch (true) {
+            case $adapter instanceof \League\Flysystem\Adapter\Local:
                 $path = $adapter->applyPathPrefix($path); // get the full path
                 $permissions = substr(sprintf('%o', fileperms($path)), -3);
                 return $permissions;
