diff --git a/src/functions.php b/src/functions.php
index 9133123..9aef0bf 100644
--- a/src/functions.php
+++ b/src/functions.php
@@ -32,7 +32,7 @@ if (! function_exists('DI\create')) {
      * @param string|null $className Class name of the object.
      *                               If null, the name of the entry (in the container) will be used as class name.
      */
-    function create(string $className = null) : CreateDefinitionHelper
+    function create(?string $className = null) : CreateDefinitionHelper
     {
         return new CreateDefinitionHelper($className);
     }
@@ -45,7 +45,7 @@ if (! function_exists('DI\autowire')) {
      * @param string|null $className Class name of the object.
      *                               If null, the name of the entry (in the container) will be used as class name.
      */
-    function autowire(string $className = null) : AutowireDefinitionHelper
+    function autowire(?string $className = null) : AutowireDefinitionHelper
     {
         return new AutowireDefinitionHelper($className);
     }
